components/mediation-monitor/mediation-data-publisher/org.wso2.carbon.das.data.publisher.util/src/main/java/org/wso2/carbon/das/data/publisher/util/DASDataPublisherConstants.java

```diff
@@ -30,4 +30,14 @@
     public static final String FLOW_STATISTIC_ANALYTICS_PUBLISHING = STAT_CONFIG_ELEMENT + ".AnalyticPublishingDisable";
     public static final String STAT_OBSERVERS = STAT_CONFIG_ELEMENT + ".Observers";
     public static final String FLOW_STATISTIC_NODE_HOST_NAME = STAT_CONFIG_ELEMENT + ".NodeHostName";
+
```

Comment:
Shall we rename the class 'DASDataPublisherConstants' since, DAS is not in use now? MAy be to AnalyticPublisherConstants?

Reply:
+1. Will Change.